pom.xml

```diff
@@ -6,7 +6,7 @@
   <groupId>ome</groupId>
   <artifactId>pom-omero-client</artifactId>
   <packaging>pom</packaging>
-  <version>5.4.9</version>
+  <version>5.5.0-SNAPSHOT</version>
```

Comment:
Worth bumping to `5.5.0-m1` to start consuming it downstream. Or do you want to test things with `-SNAPSHOT` first?

Reply:
I could try with downloader if I knew how to set the ome.maven repository to fetch snapshots.

Reply:
Assuming `omero-downloader` consumes https://artifacts.openmicroscopy.org/artifactory/maven/, both the release as well as the SNAPSHOT versions should be available.

Reply:
> Could not find artifact ome:pom-omero-client:pom:5.5.0-SNAPSHOT in ome.maven (https://artifacts.openmicroscopy.org/artifactory/maven)

Reply:
Huh yes... this PR needs to be fixed and merged first (the deployment happens via Travis)

Reply:
@mtbc now available at https://artifacts.openmicroscopy.org/artifactory/maven/ome/pom-omero-client/5.5.0-SNAPSHOT/

Reply:
@sbesson: Seems to be working fine with downloader pulling from eel, :+1:

Reply:
```diff
--- a/pom.xml
+++ b/pom.xml
@@ -9,7 +9,7 @@
   <parent>
     <groupId>ome</groupId>
     <artifactId>pom-omero-client</artifactId>
-    <version>5.4.9</version>
+    <version>5.5.0-SNAPSHOT</version>
   </parent>
```